config/default.yml

```
@@ -215,6 +215,11 @@ Performance/RedundantSortBlock:
   Enabled: 'pending'
   VersionAdded: '1.7'
 
+Performance/RedundantSplitRegexpArgument:
+  Description: 'This cop identifies places where `split` argument can be replaced from a deterministic regexp to a string.'
+  Enabled: pending
+  VersionAdded: '1.9'
```

RuboCop Performance 1.9 series has been released and a new cop will be added with minor version upgrade. I will release RuboCop Performance 1.9.1 (bugfix version) ASAP, so I will revisit it after that. Thank you for your patience.

---

Can you update to `1.10`?

```suggestion
  VersionAdded: '1.10'
```